pkg/cloud/alibaba/alibaba.go

```diff
@@ -42,9 +43,10 @@ func (a *alibabaAssets) GetRenderedResources() []client.Object {
 	return a.renderedResources
 }
 
-func getTemplateValues(images *imagesReference) (common.TemplateValues, error) {
+func getTemplateValues(images *imagesReference, operatorConfig config.OperatorConfig) (common.TemplateValues, error) {
 	values := common.TemplateValues{
-		"images": images,
+		"images":            images,
+		"cloudproviderName": operatorConfig.GetPlatformNameString(),
```

Comment:
what if the platform name is an empty string?

Reply:
Validation will fail